draft-kuhn-quic-bdpframe-extension-01.xml

```diff
@@ -127,6 +127,13 @@
 			BDP_FRAME parameters; 
 			</t>
 			<t>
+			The client can read most of the BDP_FRAME parameters but not 
+			modify them. The sender computes a secured hash with 
```

Comment:
A client can read some of the BDP_FRAME parameters but not modify any of them.

Reply:
Is it a /client/ or a /receiver/ in the context of a BDP frame?

Reply:
Fixed in commit https://github.com/NicoKos/resume_connection/commit/b6cf8df1573405b060e3cfcb62b1e0efcb38146d